hasher-matcher-actioner/hmalib/lambdas/pdq/pdq_matcher.py

```diff
@@ -116,6 +120,7 @@ def lambda_handler(event, context):
                     signal_id,
                     metadata["source"],
                     metadata["hash"],
+                    metadata["privacy_groups"],
```

Comment:
[Diff 3/5] `PDQSignalMetadata` has this information and can be looked up (we can add a helper to the class to make it easier).

Is your concern about retaining this info at all or at the very moment of the match?
- If at all I believe it is covered. 
- If at the very moment of the match then sure (Though I don't love the likely storage overlap and the different sources of truth to pick between presenting on the detail page but we can handle it) 

The fact that a signal is in a PG is not context the match record itself needs to care about imo; my understanding is that this is something `PDQSignalMetadata` and the match message would handle.

[not sure] but even if it is a at the very moment concern the match message downstream might be a better place to record this.

Reply:
You are right. This information is already present in PDQSignalMetadata. I'll rework to change how the information is inferred.